### ISSUE TITLE
Emoji-keyboard not running

### ISSUE THREAD

**Issue by Breadleaf** [OzymandiasTheGreat/emoji-keyboard#65]
I installed the package from the aur, however it will not launch when I try to run it from the command line, instead it sits at the command line and gives no errors. Once I stop it from running it produces the error:

Traceback (most recent call last):
  File "/usr/bin/emoji-keyboard", line 16, in <module>
    shared.command = lib.Command()          # Parse command line flags and decide
  File "/usr/lib/python3.9/site-packages/emoji_keyboard/emoji_lib.py", line 89, in __init__
    shared.connection.send('show_keyboard')
  File "/usr/lib/python3.9/site-packages/emoji_keyboard/emoji_lib.py", line 171, in send
    connection = Client(self.address, authkey=self.auth)
  File "/usr/lib/python3.9/multiprocessing/connection.py", line 507, in Client
    c = SocketClient(address)
  File "/usr/lib/python3.9/multiprocessing/connection.py", line 635, in SocketClient
    s.connect(address)
    
    edit: clarity on issue

**Comment by OzymandiasTheGreat** [comment]
I can't really support packages other than the AppImage. Could you run the AppImage and see if it works?
If not, post python log from `~/.config/emoji-keyboard/python.log`.

**Comment by Breadleaf** [comment]
Hi, thank you for your response. Sorry, it took me so long to respond. I just tried the AppImage and it didn't produce a log in python.log. I took some screenshots of an error it produced once I ran it. 

This one I ran ./emoji-keyboard-3.1.1.AppImage
![emoji-keyboard-error](https://user-images.githubusercontent.com/62030791/110725504-90591200-81cc-11eb-9fdc-2badeabf6893.png)

This one I ran ./emoji-keyboard-3.1.1.AppImage --no-sandbox
![emoji-keyboard-error-no-sandbox](https://user-images.githubusercontent.com/62030791/110725506-90f1a880-81cc-11eb-8491-56063141d355.
